variables.tf

```diff
@@ -23,6 +23,11 @@ variable "skip_region_validation" {
 
 
 // ECS instance
+variable "allocate_public_ip" {
+  description = "Whether to allocate public ip."
```

Comment:
Whether to allocate public ip for ECS instance. If 'create_slb' is true, it will be ignore.